src/app/services/knowledgeapi.service.ts

```diff
@@ -14,6 +14,7 @@ export class KnowledgeapiService {
   searchURL = this.server + '/w/api.php?';
   homepage = 'http://susper.com';
   logo = '../images/susper.svg';
+  imgUrl = 'https://api.duckduckgo.com/?';
```

Comment:
Is it sure, that it will return some image for every query? and please remove ```/?``` at the end of URL, it its not required.

Reply:
If the results will be present on duckduckgo it will return URL of image. `/?` is needed in the URL. This is the API from which I am fetching the results https://api.duckduckgo.com/?q=india&format=json&pretty=1